src/main/scala/SerialAdapter.scala

```diff
@@ -462,7 +508,8 @@ class MultiClockSerialAXIRAM(
   }
 
   val memXbar = memClkRstDomain { TLXbar() }
-
+  val rom = memClkRstDomain { SerialTLROM(romParams, memParams.beatBytes) }
+  rom.node := TLFragmenter(memParams.beatBytes, p(CacheBlockBytes)) := memXbar
```

Comment:
Shouldn't the `TLFrag` be in the `memClkRstDomain`?

Reply:
Yes. Good catch